tests/conftest.py

```diff
@@ -553,3 +553,8 @@ def evaluation_dataset(meta_partitions_evaluation_files_only, store_session):
             dataset_uuid="evaluation_uuid",
             store=store_session,
         )
+
+
+@pytest.fixture
+def dataset_factory_function_scope(store_factory):  # Note, this is empty
```

Comment:
What is this used for if it is empty?

Reply:
it is not empty .it contains test dataset. removed the comment.